### PR TITLE
Update kubelet_getters.go

### PR DESCRIPTION
**Pull request by jhstianwei**
add max timeout for stat volume path

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by bart0sh** [comment]
@jhstianwei Thank you for your PR. Please [sign the CLA](https://github.com/kubernetes/community/tree/master/contributors/guide/contributor-cheatsheet#signing-the-cla) to proceed further, thanks.

**Comment by yxxhero** [comment]
@jhstianwei please fix CLA issue.

> [CLA Not Signed](https://api.easycla.lfx.linuxfoundation.org/v2/repository-provider/github/sign/18706487/20580498/119530/#/?version=2)
> 
> * [❌](https://api.easycla.lfx.linuxfoundation.org/v2/repository-provider/github/sign/18706487/20580498/119530/#/?version=2) - login: @jhstianwei . The commit ([f6b6a57](https://github.com/kubernetes/kubernetes/commit/f6b6a5739ad1d945cc957e33eff90193cb02e397)) is not authorized under a signed CLA. [Please click here to be authorized](https://api.easycla.lfx.linuxfoundation.org/v2/repository-provider/github/sign/18706487/20580498/119530/#/?version=2). For further assistance with EasyCLA, [please submit a support request ticket](https://jira.linuxfoundation.org/servicedesk/customer/portal/4).

**Comment by VirrageS** [comment]
What does this MR check?

File: pkg/kubelet/kubelet_getters.go [hunk 1]
```diff
@@ -359,8 +374,11 @@ func (kl *Kubelet) getMountedVolumePathListFromDisk(podUID types.UID) ([]string,
 	// We plan to remove this mountpoint check as a condition before deleting pods since it is
 	// not reliable and the condition might be different for different types of volumes. But it requires
 	// a reliable way to clean up unused volume dir to avoid problems during pod deletion. See discussion in issue #74650
+	ctx, cancel := context.WithTimeout(ctx, maxWaitForStatVolumePath*time.Second)
+	defer cancel()
+	
 	for _, volumePath := range volumePaths {
-		isNotMount, err := kl.mounter.IsLikelyNotMountPoint(volumePath)
+		isNotMount, err := kl.IsLikelyNotMountPoint(volumePath)
```
Comment (ffromani) [inline pkg/kubelet/kubelet_getters.go:381]: missing `ctx` argument?

File: pkg/kubelet/kubelet_getters.go [hunk 2]
```diff
@@ -40,6 +40,11 @@ import (
 	"k8s.io/kubernetes/pkg/volume/csi"
 )
 
+const (
+	// Max amount of time to stat volume path
+	maxWaitForStatVolumePath = 10 * time.Second
```
Comment (ffromani) [inline pkg/kubelet/kubelet_getters.go:45]: how was this determined?

**Comment by ffromani** [comment]
/ok-to-test

thanks for this PR! is this addressing a reported issue?

**Comment by j4ckstraw** [comment]
how about check mountinfo instead of call stat? 

we check mountinfo to detemine whether /var/lib/kubelet/pods/xxx/kubernetes~csi/mount is a mountpoint, instead of stat which may struck in syscall. refer: https://github.com/kubernetes/kubernetes/pull/74653#discussion_r260707121

**Comment by dims** [comment]
@jhstianwei can you please sign the CLA? see notes from the bot above. We can't really proceed without it.

**Pull request closed without merging**